### PR TITLE
Technical blog related to Handling Data (BigData)#233

### PR DESCRIPTION
**Pull request by SamriddhiMishra**
#233 

#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] You created a branch from `dev` branch
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] Checked for all grammatical errors in the blog.
- [x] Checked for any duplicate blog entry.
- [x] You PR is raised on `dev` branch and not on`master`.

##### Author Details for Blog Post Entry

id: Samriddhi Mishra
bio: An undergrad student pursuing BTech. Eager to learn new things.
github: SamriddhiMishra
stackoverflow: 10469993
linkedin: samriddhi-mishra
medium: samriddhi2958
twitter: samriddhi07

### CODE REVIEW

**Comment by KarenEfereyan** [comment]
Another nice one @SamriddhiMishra who run the world? Data Data 🤣🤣🤣🤣

**Comment by SamriddhiMishra** [comment]
> Another nice one @SamriddhiMishra who run the world? Data Data 🤣🤣🤣🤣

💯

**Review by puneetsingh24 — COMMENTED** [review]
Hello @SamriddhiMishra 

We found out that a few of the content on the blog is copy-pasted from other sites. As part of our code of conduct, we encourage you to create your own content in your own language.

https://www.freecodecamp.org/news/a-quick-overview-of-the-apache-hadoop-framework/
https://www.flexiant.com/2013/07/26/what-is-distributed-storage-and-why-use-it-part-2/
https://www.smartdatacollective.com/how-much-big-data-companies-make-on-internet/


Also, please use png/jpg for the cover image, gif is not supported yet

Thanks
Can you please write these parts in your own language

**Comment by SamriddhiMishra** [comment]
Yeah, I will try writing in my words but the facts about the generated can't be changed.

**Comment by SamriddhiMishra** [comment]
Thanks, @puneetsingh24 for pointing out the issues. I have changed the conflicting content, but the facts can't be changed. Although I have added the link of the blog from where I collected the facts and mentioned in the blog. Please review it again.

**Comment by SamriddhiMishra** [comment]
@puneetsingh24 Please review it. Thanks!

**Review by puneetsingh24 — APPROVED** [review]

File: content/blog/handling-cheapest-fuel-data/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Handling the Cheapest Fuel- Data"
+date: "2020-10-06"
+author: "Samriddhi Mishra"
+coverImage: "a.png"
+tags: ["Data", "BigData", "Hadoop"]
+description: "You can have data without information, but you cannot have information without data. A large amount of data generated per data, the issues in handling. How big companies manage and analyze a huge amount of data?"
```
Comment (vyasriday) [inline content/blog/handling-cheapest-fuel-data/index.md:7]: `A large amount of data generated per data, the issues in handling.` Is it the correct sentence @SamriddhiMishra

Comment (SamriddhiMishra) [reply content/blog/handling-cheapest-fuel-data/index.md]: @vyasriday corrected the typo. 
Thanks!

**Review by vyasriday — APPROVED** [review]
Thanks for updating it.

**Comment by pareek-narendra** [comment]
@SamriddhiMishra 
Thanks for your contribution to LoginRadius Engineering Portal.

We found that the same article has already been published on medium. As per our guidelines, we do not accept articles that have already been published elsewhere. So now either you can remove your article from medium and use a canonical link from LoginRadius on medium, or we will have to remove your article form our platform.  If you need help with the canonical URL part, we can help you with that.
Anyway, we are syndicate LoginRadius blogs to all famous blog sites including medium.

https://medium.com/@samriddhi2958/handling-the-cheapest-fuel-data-6042b46f1f08

**Comment by SamriddhiMishra** [comment]
@pareek-narendra Yeah, I would like to use the canonical link from LoginRadius and want help with that too.

**Comment by pareek-narendra** [comment]
@SamriddhiMishra 
Please follow these steps mentioned here: https://help.medium.com/hc/en-us/articles/360033930293-Set-a-canonical-link
And this is your article link: https://www.loginradius.com/engineering/blog/handling-cheapest-fuel-data/

Please let us know once you are done or need any further help.

**Comment by SamriddhiMishra** [comment]
@pareek-narendra I added the URL Please check
image
image

**Comment by SamriddhiMishra** [comment]
@pareek-narendra Do I need to make some other changes or it is fine now?

**Comment by pareek-narendra** [comment]
@SamriddhiMishra Thanks for making the changes, It looks good.

**Pull request merged**